kong/db/dao/plugins.lua

```diff
@@ -156,6 +156,32 @@ local function load_plugin_entity_strategy(schema, db)
   if not strategy then
     return nil, err
   end
+
+  local custom_strat = fmt("kong.db.strategies.%s.%s", db.strategy, schema.name)
```

Comment:
actually im wondering if this should something like

```
local custom_strat = fmt("kong.plugins.%s.db.strategies.%s.%s",
plugin, db.strategy, schema.name)
```
?

Reply:
I'd go without the `.db.`, since we don't use it for `migrations`:

```
local custom_strat = fmt("kong.plugins.%s.strategies.%s.%s",
plugin, db.strategy, schema.name)
```

(I think this was the path originally suggested by @thibaultcha too)

Reply:
Updated, thanks!